src/FSharpx.Collections.Experimental/BinomialHeap.fs

```diff
@@ -263,7 +263,9 @@ module BinomialHeap =
     ///O(log n). Returns heap option from merging two heaps.
     let inline tryMerge (xs: BinomialHeap<'T>) (ys: BinomialHeap<'T>) = xs.TryMerge ys
 
-    ///O(log n). Returns heap from the sequence.
+    //FIX
```

Comment:
I left the explanation because that is not my code. I just found that and fix it, and was a harmless change. You are right, I'll clean it.